### PR TITLE
Pages 4.0 plan update

### PR DESCRIPTION
**Pull request by markt-asf**
## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:
- [X] A directory in the form wombat/x.y where x.y is the release major.minor version.
- [X] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [X] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [X] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [X] Includes a detailed description of any backward incompatibility (Mark with **N/A** and check if none)
- [X] **N/A** Declares optional features if there are any (Mark with **N/A** and check if none)
- [X] Includes minimum Java SE version
- [X] **N/A** Describe any updates or required changes including splitting out the TCK (Mark with **N/A** and check if not planned)
- [X] Link to updated release record

### CODE REVIEW

**Comment by dblevins** [comment]
Hi @markt-asf 

On the "All code deprecated as of Jakarta Server Pages 3.1 has been removed"

Just trying to get an idea of impact.  Do you have a list of the code that was deprecated?  If there's a PR with the removals, that would help.  If there's any kind of theme to the deprecation, that's helpful.

Here's how we described our removals in [EJB 4.0](https://jakarta.ee/specifications/enterprise-beans/4.0/):

image

**Comment by ivargrimstad** [comment]
For 3.1, the section said:

### Removals, deprecations or backwards incompatible changes
* Deprecate methods that override ELResolver.getFeatureDescriptors() as that method has been deprecated as of EL 5.0
* Deprecate the isThreadSafe page directive attribute as the related Servlet API interface SingleThreadModel has been removed as of the Servlet 6.0 specification
* [226](https://github.com/eclipse-ee4j/jsp-api/issues/226) Deprecate the jsp:plugin action and related actions as the associated HTML elements are no longer supported by any major browser

So basically what David is asking is to repeat this list on the 3.2 pages, but with the text "Deprecate" replaced with "Removed"

**Comment by dblevins** [comment]
Thanks @markt-asf and @ivargrimstad.  This looks good to me.
